coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -197,3 +197,22 @@ def from_metadata(cls, doccomment, docstyle_definition,
 
         return DocumentationComment(assembled_doc, docstyle_definition, indent,
                                     marker, range)
+
+    def assemble(self):
+        """
+        Assembles parsed documentation to the original documentation.
+
+        This function assembles the whole documentation comment, with the
+        given markers and indentation.
+        """
+        lines = self.documentation.splitlines(keepends=True)
+        assembled = self.indent + self.marker[0]
+        if len(lines) == 1:
+            assembled += lines[0]
+        else:
+            assembled += ''.join('\n' if line == '\n' and not self.marker[1]
+                                 else self.indent + self.marker[1] + line
+                                 for line in lines)
```

Comment:
I just found about this function:
https://docs.python.org/3.4/library/textwrap.html#textwrap.indent

maybe you can use this one :)
so you just need to generate the stuff without indentation and make the indent special only for the start-marker and end-marker (if it's that easier, I don't know :3)